src/Actions/KanbanData/GetKanbanDataAction.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Xguard\LaravelKanban\Actions\KanbanData;
+
+use DateTime;
+use Exception;
+use Lorisleiva\Actions\Action;
+use Xguard\LaravelKanban\Http\Helper\AccessManager;
+use Xguard\LaravelKanban\Models\Board;
+use Xguard\LaravelKanban\Models\Log;
+
+class GetKanbanDataAction extends Action
+{
+    public function authorize(): bool
+    {
+        return AccessManager::canAccessBoardUsingTaskId($this->boardId);
+    }
+
+    protected function failedAuthorization()
+    {
+        abort(403, "You don't have access to this board");
```

Comment:
Siamak you use magic numbers, please use constants

Reply:
oki dokie artichokie